api/src/main/java/ca/bc/gov/educ/api/trax/config/GradTraxConfig.java

```diff
@@ -55,13 +64,15 @@ public WebClient getTraxClientWebClient(OAuth2AuthorizedClientManager authorized
 		ServletOAuth2AuthorizedClientExchangeFilterFunction filter = new ServletOAuth2AuthorizedClientExchangeFilterFunction(authorizedClientManager);
 		filter.setDefaultClientRegistrationId("traxclient");
 		return WebClient.builder()
+				.filter(setRequestHeaders())
 				.exchangeStrategies(ExchangeStrategies
 						.builder()
 						.codecs(codecs -> codecs
 								.defaultCodecs()
 								.maxInMemorySize(50 * 1024 * 1024))
 						.build())
 				.apply(filter.oauth2Configuration())
+				.filter(this.log())
 				.build();
 	}
 
```

Comment:
log() and setRequestHeaders() need to be on _all_ WebClient implementations. You missed instituteWebClient.